apps/measures/models.py

```diff
@@ -518,9 +518,29 @@ class MeasuresDetailPage(Page):
     ]
 
     @property
-    def get_fields_of_action_display(self):
-        fields_of_action = [self.get_regenerative_management_display(),
-                            self.get_future_opportunities_for_all_display(),
-                            self.get_inclusive_shaping_of_urban_life_display(),
-                            self.get_facilitative_administration_display()]
-        return ', '.join(filter(None, fields_of_action))
+    def get_fields_of_action(self):
+        fields_of_action = self.get_selected_foa(
```

Comment:
Could you add a docstring sth. like
`Return all selected fields_of_action in the form ....`
or `Return a dictianary of all selected measures and their sets`